metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -197,7 +202,12 @@ func (sink *StackdriverSink) ExportData(dataBatch *core.DataBatch) {
 		requests = append(requests, req)
 	}
 
-	go sink.sendRequests(requests)
+	select {
+	case sink.batchesQueue <- requests:
+		// queue not full, batch will be processed
+	default:
```

Comment:
FYI In case of blocking for longer period this implementation will not preserve last 5 batches, but first 5 from when outage happen.
I'm ok with it as changing this behavior would complicate code